frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -22,6 +21,15 @@ const CONSUMERS_CPU_QUERY = 'sort(topk(10, sum by (pod_name)(container_cpu_usage
 const CONSUMERS_MEMORY_QUERY = 'sort(topk(10, sum by (pod_name)(container_memory_usage_bytes{pod_name!=""})))';
 const OPENSHIFT_VERSION_QUERY = 'openshift_build_info{job="apiserver"}';
 
+const CAPACITY_CPU_TOTAL_QUERY = 'sum(kube_node_status_capacity_cpu_cores)';
+const CAPACITY_CPU_USED_QUERY = 'sum(kube_node_status_capacity_cpu_cores) - sum(kube_node_status_allocatable_cpu_cores)';
+const CAPACITY_MEMORY_TOTAL_QUERY = 'sum(kube_node_status_capacity_memory_bytes)/(1024*1024*1024)'; // in GB
```

Comment:
not sure how this query is computed but maybe it would make sense to make variables out of `1024*1024*...`

Reply:
this goes to prometheus, I would not complicate things ...

Reply:
ok

Reply:
We should get the data in bytes and then convert it to appropriate units in an intelligent way. Something like this utility function
```
export const formatBytes = (bytes, fixed = 2) => {
  if(bytes < 1024) return bytes + " Bytes";
  else if(bytes < 1024**2) return(bytes / 1024).toFixed(fixed) + " KB";
  else if(bytes < 1024**3) return(bytes / 1024**2).toFixed(fixed) + " MB";
  else if(bytes < 1024**4) return(bytes / 1024**3).toFixed(fixed) + " GB";
  else if(bytes < 1024**5) return(bytes / 1024**4).toFixed(fixed) + " TB";
  else return(bytes / 1024**5).toFixed(2) + " PB";
};
```

Reply:
Which I also plan to use for Top Consumers card :)

Reply:
I think that is a valid point. We should show the units inside the card depending on the value.

Reply:
Done even for non-storage units.
We need to keep units for used/total/available consistent.